File: customercenter/src/main/java/cafeteria/Mypage.java
package cafeteria;

import javax.persistence.*;
import java.util.List;

@Entity
@Table(name="Mypage")
public class Mypage {

        @Id
        @GeneratedValue(strategy=GenerationType.AUTO)
        private Long id;
        private Long orderId;
        private String productName;
        private Integer qty;
        private Integer amt;
        private String status;


        public Long getId() {
            return id;
        }

        public void setId(Long id) {
            this.id = id;
        }
        public Long getOrderId() {
            return orderId;
        }

        public void setOrderId(Long orderId) {
            this.orderId = orderId;
        }
        public String getProductName() {
            return productName;
        }

        public void setProductName(String productName) {
            this.productName = productName;
        }
        public Integer getQty() {
            return qty;
        }

        public void setQty(Integer qty) {
            this.qty = qty;
        }
        public Integer getAmt() {
            return amt;
        }

        public void setAmt(Integer amt) {
            this.amt = amt;
        }
        public String getStatus() {
            return status;
        }

        public void setStatus(String status) {
            this.status = status;
        }

}

File: customercenter/src/main/scala/cafeteria/Mypage.scala
package cafeteria

import scala.beans.BeanProperty
import org.springframework.data.mongodb.core.mapping.Document
import org.springframework.data.annotation.Id
import org.springframework.data.annotation.Transient

object Mypage {
  val SEQUENCE_NAME :String = "mypage_sequence"  
}

@Document
class Mypage {
  
  @Id
  @BeanProperty
  var id :Long = 0L
  
  @BeanProperty
  var orderId :Long = 0L
  
  @BeanProperty
  var productName :String = null
  
  @BeanProperty
  var qty :Integer = 0
  
  @BeanProperty
  var amt :Integer = 0
  
  @BeanProperty
  var phoneNumber :String = null
  
  @BeanProperty
  var status :String = null
}
